service/src/test/java/uk/gov/hmcts/reform/fpl/utils/LegalCounsellorTestHelper.java

```diff
@@ -1,59 +1,22 @@
 package uk.gov.hmcts.reform.fpl.utils;
 
-import org.apache.commons.lang3.tuple.Pair;
 import uk.gov.hmcts.reform.ccd.model.Organisation;
 import uk.gov.hmcts.reform.fpl.model.LegalCounsellor;
-import uk.gov.hmcts.reform.fpl.service.OrganisationService;
-
-import java.util.Optional;
-
-import static org.mockito.Mockito.when;
 
 public class LegalCounsellorTestHelper {
 
     private LegalCounsellorTestHelper() {
         //NO-OP
```

Comment:
What is this for?

Reply:
not sure but it was being used and needed updating

Reply:
just seems to be there for generating test legal counsellors